core/arch/riscv/kernel/boot.c

```diff
@@ -66,6 +66,7 @@ void init_tee_runtime(void)
 
 static void init_primary(unsigned long nsec_entry)
 {
+	thread_init_core_local_stacks();
```

Comment:
nitpicking: an empty line below would be great.

Reply:
done